spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'User index page', type: :feature do
+  before :each do
+    @user1 = User.create(name: 'Tomas', photo: 'https://usp.com/pto.png', bio: 'Web developer')
+    visit root_path
+  end
+
+  it 'displays username of all other users' do
+    expect(page).to have_content(@user1.name)
```

Comment:
- [ ] It would be better if we can display the user name for all other users. Currently, we are displaying just one user. But we can fix this by `all_user = User.all` then we can do `expect(page).to have_content(all_user.name)`. Kindly try this out

Reply:
Hello @addod19, Thanks for your feedback. However, I don't think the method you propose here works as you hope it would. Using all_data.name only returns 'User'. Besides, I have data in my `before `block for the testing.
![Screenshot 2022-07-19 204309](https://user-images.githubusercontent.com/8092327/179844765-047b4b12-2cac-423a-9ca5-a1610b56d3bd.png)